tutorials/migrating/upgrading_to_godot_4.3.rst

```diff
@@ -0,0 +1,356 @@
+
+Behavior changes
+----------------
+
+In 4.3 some behavior changes have been introduced, which might require you to adjust your project.
+
+Core
+^^^^
+
+.. note::
+
+    Binary serialization was modified to fix some issues with the serialization of scripted Objects and typed Arrays (`GH-78219`_).
+    This breaks compat with script encoding/decoding.
+
+.. note::
+
+    ``PackedByteArray`` is now able to use a more compact base64 encoding for storage. But the trade-off is that it breaks
+    compatibility, meaning that older versions of Godot may not be able to open resources saved by 4.3. (`GH-89186`_).
+
+    To maximize compatibility, this new storage format will only be enabled for resources and scenes that contain large
+    PackedByteArrays for now. Support for this new format will also be added in patch updates for older versions of Godot.
+    Once all supported Godot versions are able to read the new format, we will gradually retire the compatibility measures
+    and have all resources and scenes use the new storage format.
+
+.. note::
+
+    In C#, the ``Transform3D.InterpolateWith`` implementation was fixed to use the right order of operations, applying the rotation before the scale (`GH-89843`_).
+
+.. note::
+
+    In C#, the ``Aabb.GetSupport`` implementation was fixed to properly return the support vector (`GH-88919`_).
+
+.. note::
+
+    In C#, the Variant types' ``ToString`` implementation now defaults to using the ``InvariantCulture`` (`GH-89547`_)
+    which means ``Vector2(1.2, 3.4)`` is formatted using ``.`` as the decimal separator independently of the language
+    of the operating system that the program is running on.
+
+Animation
+^^^^^^^^^
+
+.. note::
+
+    ``AnimationMixer`` replaced its Capture mode with a new Capture feature that works much better than the old one,
+    this replaces the existing cache (`GH-86715`_).
+
+.. note::
+
+    ``AnimationNode`` has a reworked process for retrieving the semantic time info. This ensures that time-related
+    behavior works as expected, but changes the blending behavior. Implementors of the ``_process`` virtual method
+    should also note that this method is now deprecated and will be replaced by a new one in the future (`GH-87171`_).
+
+GUI nodes
+^^^^^^^^^
+
+.. note::
+
+    The default font outline color was changed from white to black. (`GH-54641`_).
+
+.. note::
+
+    The ``auto_translate`` property is deprecated in favor of the ``auto_translate_mode`` property which is now in ``Node`` (`GH-87530`_).
+    The default value for ``auto_translate_mode`` is ``AUTO_TRANSLATE_INHERIT``, which means nodes inherit the ``auto_translate`` value
+    from their parent. This means, existing nodes with the ``auto_translate`` property set to ``true`` may no longer be translated if they
```

Comment:
```suggestion
    The default value for ``auto_translate_mode`` is ``AUTO_TRANSLATE_INHERIT``, which means nodes inherit the ``auto_translate_mode`` value
    from their parent. This means, existing nodes with the ``auto_translate`` property set to ``true`` may no longer be translated if they
```

Reply:
The https://github.com/godotengine/godot/pull/87530 PR description says:

> `AUTO_TRANSLATE_MODE_INHERIT` - The new default, inherits the `auto_translate` value of its parent.

Is this incorrect? It sounds like inheriting the `auto_translate` value is the reason why this breaks compat.

Reply:
Internally `auto_translate` doesn't exist anymore, it just pipes into `auto_translate_mode` when it's changed/checked.